lms/services/hapi.py

```diff
@@ -0,0 +1,115 @@
+"""Hypothesis API service."""
+import json
+
+import requests
+from requests import RequestException
+
+from lms.services import HAPIError
+
+
+class HypothesisAPIService:
+    """
+    Low-level API service.
+
+    A service that provides low-level methods for calling the Hypothesis API
+    with builtin error handling. This service provides HTTP verb methods like
+    ``get()`` and ``post()``. Other services can be built on top of this one
+    and provide higher-level methods like ``create_user()`` or
+    ``update_group()``.
+    """
+
+    def __init__(self, _context, request):
+        """
+        Return a new HypothesisAPIService object.
+
+        :arg _context: the Pyramid context resource
```

Comment:
The `_context` argument is required by Pyramid but isn't used by our class. Prefixing a required argument with `_` prevents pylint from complaining about "unused argument".